packages/expo-device/CHANGELOG.md

```diff
@@ -12,6 +12,8 @@
 
 ### 💡 Others
 
+- [iOS] Added some missing entries for 2024 Apple devices. ([#31970](https://github.com/expo/expo/pull/31970) by [@fobos531](https://github.com/fobos531))
```

Comment:
In theory it's a new feature 😄

Reply:
I was debating whether to put it in Others or New features haha - moved